### DISCUSSION TITLE
Reorganization has made a mess of Insomnia. All environments lost for all requests.

### DISCUSSION THREAD

**Discussion by sanderheuver66** [Kong/insomnia#5560] [Feedback]
I have been using Insomnia for about 5 years now. I used to be a fan of Insomnia and I have persuaded other people to use it. But this last update has ruined my day, maybe even my week.  I used to have about 30 to 40 requests, organized in about 5 or 6 folders. Two environments for each of the requests (test and production).

Update 2022.7.0 occurred and now? My left column only contains a Project called "Insomnia". My folders and requests have been reorganized into 19 "Files", all of type(?) Collection, each of which contains a number of my original requests. I have to guess where a request is located.

All requests seem to have lost their environments. Sometimes one is visible, but usually "No Environments".

I have been able to retrieve the Test and Production settings from the environments db file, but now I have to edit all requests and add them there. I don't see a feature where I can manage environments for all my requests. And how do I move requests from one File/Collection to another? What a mess!

**Comment by aidarchikable** [comment]
I have the same issue. I was using Insomnia as a replacement for Postman. but now Insomnia did lose all my collections and changed UI the same way as Postman.

**Reply by choffmanebpm** [reply]
I completely lost everything as of this morning as well.

**Comment by HoganDeckis** [comment]
I'm uninstalling and going back to postman after this update.

**Comment by j-tomas** [comment]
Does anyone know how to recover lost data?

**Comment by ericdrosas87** [comment]
I just wanted to chime in that the update basically ruins what was great about Insomnia: the simplicity. Switching back to PostMan myself.

**Comment by davidbonnet** [comment]
What a terrible mess. How is this even possible to allow such an update without warning users?

**Comment by ozzyfromspace** [comment]
Conspiracy: Insomnia was secretly sponsored by Postman. The latest update made everything so convoluted.

**Comment by jasonatr0n** [comment]
This has happened to me more than once. It's terrible design. I am NOT paying anything for this. Sorry, but I tried.
